straxen/contexts.py

```diff
@@ -59,7 +60,9 @@
 
 def xenonnt_online(output_folder='./strax_data',
                    we_are_the_daq=False,
+                   _minimum_run_number=9271,
```

Comment:
@WenzDaniel , this would only affect the `st.select_runs` you can still use st.get_array if the data is stored (with the right lineage) in one of the frontends.